apps/ledger-live-desktop/src/renderer/modals/Send/fields/AmountField.tsx

```diff
@@ -35,6 +36,7 @@ const AmountField = ({
   initValue,
   resetInitValue,
   walletConnectProxy,
+  useMaxLabel,
```

Comment:
`use*` variable naming is a convention only reserved to hooks, consider renaming this to maybe `enableMaxLabel` or `withMaxLabel` or `maxLabelEnabled`